packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -252,21 +260,25 @@ export type ISummaryConfiguration =
 export const DefaultSummaryConfiguration: ISummaryConfiguration = {
     state: "enabled",
 
-    idleTime: 5000 * 3,
+    idleTime: 15 * 1000, // 15 secs.
 
-    maxTime: 5000 * 12,
+    maxTime: 60 * 1000, // 1 min.
 
-    maxOps: 1000, // Summarize if 1000 ops received since last snapshot.
+    maxOps: 100, // Summarize if 100 weighted ops received since last snapshot.
 
     minOpsForLastSummaryAttempt: 10,
 
-    maxAckWaitTime: 6 * 10 * 1000, // 6 min.
+    maxAckWaitTime: 10 * 60 * 1000, // 10 mins.
 
     maxOpsSinceLastSummary: 7000,
 
-    initialSummarizerDelayMs: 5000, // 5 secs.
+    initialSummarizerDelayMs: 5 * 1000, // 5 secs.
 
     summarizerClientElection: false,
+
+    systemOpWeight: 0.1,
+
+    nonSystemOpWeight: 1.0,
```

Comment:
Should we call it user op weight?
I think we plan to use "runtime" word - see https://github.com/microsoft/FluidFramework/issues/9732

Reply:
Let's find a way to use same terminology everywhere.

Reply:
Are you suggesting to use `clientOpWeight` and `runtimeOpWeight` instead of `systemOpWeight` and `nonSystemOpWeight` respectively?

Reply:
I think it's fine to use runtime and non-runtime pair, or runtime & system pair of words. While later does not exactly map to actual terms, I think it's fine.